README.md

```diff
@@ -316,16 +321,114 @@ Alongside the built assets a metadata file will be created with info about the b
         "opt": "202007070926"
     },
     "scmRef": "<output of git describe OR buildConfig.SCM_REF>",
+    "buildRef": "<build-repo-name/build-commit-sha>",
     "version_data": "jdk15",
     "binary_type": "debugimage",
     "sha256": "<shasum>",
     "full_version_output": <output of java --version>,
     "configure_arguments": <output of bash configure>
 }
 ```
+The Metadata class is contained in the [Metadata.groovy](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/library/src/common/MetaData.groovy) file and the Json is constructed and written in the [openjdk_build_pipeline.groovy](https://github.com/AdoptOpenJDK/openjdk-build/blob/master/pipelines/build/common/openjdk_build_pipeline.groovy) file.
 
 It is worth noting the additional tags on the semver is the adopt build number.
 
+- `vendor:`
+Example values: [`AdoptOpenJDK`, `Alibaba`]
+
+This tag is used to identify the vendor of the JDK being built, this value is set in the [build.sh](https://github.com/AdoptOpenJDK/openjdk-build/blob/805e76acbb8a994abc1fb4b7d582486d48117ee8/sbin/build.sh#L183) file and defaults to "AdoptOpenJDK".
+
+- `os:`
+Example values: [`windows`, `mac`, `linux`, `aix`, `solaris`]
+
+This tag identifies the operating system the JDK has been built on (and should be used on).
+
+- `arch:`
```

Comment:
Missing explanation (to keep it the same as the others)

Reply:
Added missing archs and a small description.